remote-config.jsonc

```diff
@@ -123,6 +123,30 @@
         },
         {
           "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
+        },
+        {
+          "message": "You can contribute your own tips to these daily tips with a PR on https://github.com/ddev/remote-config"
+        },
+        {
+          "message": "DDEV's goal is to make developers happy. We hope you're happy today!"
+        },
+        {
+          "message": "If you're on macOS you may want to try the excellent new Docker provider OrbStack, https://orbstack.dev"
+        },
+        {
+          "message": "Most teams check in the project .ddev directory so everybody has the same configuration. But do NOT force-add the .ddev/.gitignore, which is managed by DDEV."
+        },
+        {
+          "message": "DDEV has support for many database browsers. Newest is `ddev dbeaver` but see all the others at https://ddev.readthedocs.io/en/latest/users/usage/database-management/#database-guis"
+        },
+        {
+          "message": "Today is the first day of the rest of your life! Use it well :)"
```

Comment:
This one is kind of random. Feels like something one will read from a fortune cookie lol.

Reply:
Exactly. It's a little bit of fun inside a world of serious contemplation :)

Reply:
```suggestion
          "message": "Believe you can, and you’re halfway there. – Theodore Roosevelt"
```

Reply:
It begs the question of what the goal should be for these messages, because someone may enjoy getting tips about DDEV but find a totally unrelated motivational quote to be an obstacle to that enjoyment. But I could also just be no fun.

Reply:
Somebody suggested an occassional random message like this at Drupalcon. I kind of think just one-in-a-million is ok. Might make somebody smile. It wasn't something I would have thought of either. We're all too serious.